dvr_scan/scanner.py

```diff
@@ -45,7 +45,8 @@ class ScanContext(object):
     which includes application initialization, handling the options,
     and coordinating overall application logic (via scan_motion()). """
 
-    def __init__(self, args):
+    def __init__(self, args, updateGUI=None):
```

Comment:
Is this argument necessary?

Reply:
No i checked it. It's a leftover from a different approach to updating the GUI.